h2o-algos/src/test/java/hex/kmeans/KmeansConstrainedTest.java

```diff
@@ -259,4 +260,34 @@ public void testWeatherChicagoConstrained() {
         }
     }
 
+    @Test @Ignore
```

Comment:
why ignored?

Reply:
It uses big data, and it is really slow. I used it for internal testing. Should I remove it?

Reply:
since it can be useful for development, I would just add a comment

Reply:
it is fine to keep, however, usually these things go bad over time - many of the Ignored tests don't work anymore

maybe (and really just an idea) - could we turn it into a benchmark - since you are still improving performance?